charts/images.yaml

```
@@ -0,0 +1,29 @@
+images:
+- name: csi-driver-lvm
+  sourceRepository: https://github.com/metal-stack/csi-driver-lvm
+  repository:  ghcr.io/metal-stack/csi-driver-lvm
+  tag: "v0.6.0"
+- name: csi-driver-lvm-provisioner
+  sourceRepository: https://github.com/metal-stack/csi-driver-lvm
+  repository:  ghcr.io/metal-stack/csi-driver-lvm-provisioner
+  tag: "v0.6.0"
+- name: csi-attacher
+  sourceRepository: https://github.com/kubernetes-csi/external-attacher
+  repository:  k8s.gcr.io/sig-storage/csi-attacher
+  tag: "v3.5.0"
+- name: livenessprobe
+  sourceRepository: https://github.com/kubernetes-csi/livenessprobe
+  repository:  k8s.gcr.io/sig-storage/livenessprobe
+  tag: "v2.7.0"
+- name: csi-provisioner
+  sourceRepository: https://github.com/kubernetes-csi/external-provisioner
+  repository:  k8s.gcr.io/sig-storage/csi-provisioner
+  tag: "v3.2.1"
+- name: csi-node-driver-registrar
+  sourceRepository: https://github.com/kubernetes-csi/node-driver-registrar
+  repository:  k8s.gcr.io/sig-storage/csi-node-driver-registrar
+  tag: "v2.5.1"
+- name: csi-resizer
+  sourceRepository: https://github.com/kubernetes-csi/external-resizer
+  repository:  k8s.gcr.io/sig-storage/csi-resizer
+  tag: "v2.5.1"
```

I think it should be:

```suggestion
  tag: "v1.6.0"
```

https://github.com/metal-stack/helm-charts/blob/v0.4.14/charts/csi-driver-lvm/values.yaml#L37